.github/workflows/ci.yml

```diff
@@ -81,12 +76,16 @@ jobs:
       submodules: false
       coverage: codecov
       toxdeps: tox-pypi-filter
-      posargs: -n auto
+      posargs: -n auto --dist loadgroup
```

Comment:
This is only needed for online tests.

Reply:
Is it? I will remove it.

Reply:
It is to reduce the chance that the email address makes multiple requests to JSOC before waiting for it to be fulfilled. (By running that group of tests in serial.)